src/Tasks.UnitTests/WriteCodeFragment_Tests.cs

```diff
@@ -426,6 +426,30 @@ public void OneAttributeTwoPositionalParams()
             File.Delete(task.OutputFile.ItemSpec);
         }
 
+        [Fact]
+        public void OneAttributeTwoSamePositionalParams()
```

Comment:
```suggestion
        public void OneAttributeTwoPositionalParamsWithSameValue()
```

?

Reply:
naming things 😢  I'll think of a better one.